resolwe_bio/processes/reads_processing/trimmomatic.yml

```diff
@@ -131,193 +149,281 @@
+            needed to perform Illuminacliping.
         - name: simple_clip_threshold
           label: Simple clip threshold
           type: basic:integer
           required: false
           description: >
             Specifies how accurate the match between any adapter etc. sequence must be against a read.
+            This field as well as 'Adapter sequence', 'Seed mismatches' and 'Palindrome clip threshold' parameters are
+            needed to perform Illuminacliping
         - name: palindrome_clip_threshold
           label: Palindrome clip threshold
           type: basic:integer
           required: false
           description: >
             Specifies how accurate the match between the two 'adapter ligated' reads must be for PE palindrome read alignment.
+            This field as well as 'Adapter sequence', 'Simple clip threshold' and 'Seed mismatches' parameters are
+            needed to perform Illuminacliping
         - name: min_adapter_length
-          label: Min adapter length
+          label: Minimum adapter length
           type: basic:integer
+          disabled: >
+            !illuminaclip.seed_mismatches &&
+            !illuminaclip.simple_clip_threshold &&
+            !illuminaclip.palindrome_clip_threshold
           required: false
           description: >
-             In addition to the alignment score, palindrome mode can verify that a minimum length of adapter has been detected. If unspecified, this defaults to 8 bases, for historical reasons. However, since palindrome mode has a very low false positive rate, this can be safely reduced, even down to 1, to allow shorter adapter fragments to be removed.
+             In addition to the alignment score, palindrome mode can verify that a minimum length of adapter has been
+             detected. If unspecified, this defaults to 8 bases, for historical reasons. However, since palindrome mode
+             has a very low false positive rate, this can be safely reduced, even down to 1, to allow shorter adapter
+             fragments to be removed. This field is optional for preforming Illuminaclip. 'Adapter sequence', 'Simple clip
+             threshold', 'Seed mismatches' and 'Palindrome clip threshold' are also needed in order to use this parameter.
         - name: keep_both_reads
           label: Keep both reads
           type: basic:boolean
+          disabled: >
+            !illuminaclip.seed_mismatches &&
+            !illuminaclip.simple_clip_threshold &&
+            !illuminaclip.palindrome_clip_threshold &&
+            !illuminaclip.min_adapter_length
           required: false
           description: >
-            By specifying 'true' for this parameter, the reverse read will also be retained, which may be useful e.g. if the downstream tools cannot handle a combination of paired and unpaired reads.
+            By specifying this parameter, the reverse read will also be retained, which may be useful e.g.
```

Comment:
The description provided here is not sufficient. The earlier sentences in the manual explain well what this parameter does. Add additional description here.